python/docs/python.rst

```diff
@@ -0,0 +1,9 @@
+Welcome to the rmm Python documentation!
+========================================
```

Comment:
As above, this makes for an awkward breadcrumb.

![image](https://github.com/rapidsai/rmm/assets/3943761/b7969701-2800-4d87-b803-a57ef9ffbcd9)

```suggestion
rmm Python
==========
```